eZ/Bundle/EzPublishCoreBundle/Resources/config/routing/content.yml

```diff
@@ -0,0 +1,3 @@
+_ezplatform_download:
+    path: /content/download/{contentId}/{fieldIdentifier}/{downloadName}
```

Comment:
yes, i can see that. just wondering if we could avoid that and download the file exactly the same as the original filename.